Models/IVolunteerUser.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Karma.Models
+{
+    public interface IVolunteerUser
```

Comment:
I believe IVolunteer is enough. I don't think we want to have connections between volunteers and users for now
@cezaris13 do you agree?

Reply:
Yes, I agree, for now this could be like this

Reply:
Changed the name to IVolunteer